scripts/create-dev-release.sh

```diff
@@ -3,6 +3,9 @@
 set -euo pipefail
 
 export UAA_VERSION=0.0.0
+JAVA_HOME=$(/usr/libexec/java_home -v 11)
```

Comment:
Why this change? The script should not be tied with specific Java version.

Reply:
Also, this is likely to work only on MacOS. It should be reverted.

Reply:
@swalchemist Can you remove this?